samples/openthread/cli/README.rst

```diff
@@ -518,6 +519,17 @@ To test the Thread 1.2 and Thread 1.3 features, complete the following steps:
             addresses: [fdde:ad00:beef:0:e0fc:dc28:1d12:8c2]
          Done
 
+Power consumption measurements
+==============================
+
+The Thread CLI sample can be used to perform power consumption measurements for Sleepy End Devices.
```

Comment:
```suggestion
You can use the Thread CLI sample to perform power consumption measurements for Sleepy End Devices.
```
I also noticed we are using the SED acronym without defining it in this sample. Could you change Line 360 "SED" to "Sleepy End Device (SED)", I believe that's the first spot where the term is used. That should cover it.